### PR TITLE
Remove the self written profiling solution

### LINKED ISSUE #59: Remove unnecessary AOP profiling feature
We have the home grown profiling solution that is implemented as an aspect in `MeasureDurationTimeOfMethods`. It never was the most elegant solution to begin with and since we have introduced the most excellent [metrics framework](https://github.com/dropwizard/metrics) it is now time for this  implementation to go. I suggest that we remove it.

### PR DESCRIPTION
**Pull request by wallner**
Since we introduced the dropwizard Metrics framework there is no need
anymore to carry this self written solution along. Closes #59

### CODE REVIEW

File: src/main/deploy/resource-server.properties [hunk 1]
```diff
@@ -5,7 +5,6 @@ org.osiam.resource-server.db.url=jdbc:postgresql://localhost:5432/ong
 org.osiam.resource-server.db.username=ong
 org.osiam.resource-server.db.password=b4s3dg0d
 
-# OSIAM resource server configuration
 org.osiam.resource-server.profiling=false
```
Comment (dacrome) [inline src/main/deploy/resource-server.properties]: Please also remove the property

**Comment by dacrome** [comment]
Please also remove the `org.osiam.resource-server.profiling` property [in the ITs](https://github.com/osiam/connector4java-integration-tests/blob/e1468fea094709c647eb3ad12697ea6baa33c792/src/test/resources/resource-server-configuration) and [in the docs](https://github.com/osiam/resource-server/blob/master/docs/configuration.md#configuration)

File: src/main/deploy/resource-server.properties (hunk 1 above)
Comment (wallner) [inline src/main/deploy/resource-server.properties]: Oh dear!

**Comment by wallner** [comment]
I'm going to remove the property from the ITs in a different PR.

**Comment by tkrille** [comment]
BTW: just because we added Metrics doesn't mean that we already track the information about what method spent what amount of time to complete. We have to configure this first.

Nevertheless, it is OK to remove this now, as we want to introduce `spring-boot-actuator` when the migration to Spring Boot is completed.

**Comment by dacrome** [comment]
[Spring Boot Actuator](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready) doesn't support profiling in a special way, we need to do this with Metrics. Anyway this won't be like the self written aop stuff.

**Comment by dacrome** [comment]
Thanks!

**Comment by dacrome** [comment]
Please also remove the property in the [osiam docs](https://github.com/osiam/osiam/blob/master/docs/detailed-reference-installation.md#configuring-osiam).

**Pull request merged**